sa/sa_test.go

```diff
@@ -1853,6 +1853,45 @@ func TestNewAuthorizations2(t *testing.T) {
 	}
 }
 
+func TestNewAuthorizations2_100(t *testing.T) {
+	sa, fc, cleanUp := initSA(t)
+	defer cleanUp()
+
+	reg := satest.CreateWorkingRegistration(t, sa)
+	expires := fc.Now().Add(time.Hour).UTC().UnixNano()
```

Comment:
`initSA` sets `fc` as to `time.UTC` (e.g. `fc.Set(time.Date(2015, 3, 4, 5, 0, 0, 0, time.UTC))`), so this additional call of `UTC()`, while not hurting anything, isn't necessary.

```suggestion
	expires := fc.Now().Add(time.Hour).UnixNano()
```